optd-perftest/src/benchmark.rs

```diff
@@ -9,7 +10,7 @@ impl Benchmark {
     pub fn get_strid(&self) -> String {
```

Comment:
```suggestion
    pub fn get_name(&self) -> String {
```

At first glance I'm like: what does "strid" mean 🤣